app/services/analytics_events.rb

```diff
@@ -2798,32 +2798,41 @@ def telephony_otp_sent(
 
   # @param [Boolean] success
   # @param [Hash] errors
+  # @param [String] sign_up_mfa_priority_bucket
   # Tracks when the the user has selected and submitted additional MFA methods on user registration
-  def user_registration_2fa_additional_setup(success:, errors: nil, **extra)
+  def user_registration_2fa_additional_setup(success:, sign_up_mfa_priority_bucket:,
+                                             errors: nil, **extra)
     track_event(
       'User Registration: Additional 2FA Setup',
       {
         success: success,
+        sign_up_mfa_priority_bucket: sign_up_mfa_priority_bucket,
         errors: errors,
         **extra,
       }.compact,
     )
   end
 
   # Tracks when user visits additional MFA selection page
-  def user_registration_2fa_additional_setup_visit
-    track_event('User Registration: Additional 2FA Setup visited')
+  # @param [String] sign_up_mfa_priority_bucket
+  def user_registration_2fa_additional_setup_visit(sign_up_mfa_priority_bucket:, **_extra)
+    track_event(
+      'User Registration: Additional 2FA Setup visited', {
+        sign_up_mfa_priority_bucket: sign_up_mfa_priority_bucket,
+      }
```

Comment:
Curly braces optional

```suggestion
      'User Registration: Additional 2FA Setup visited',
      sign_up_mfa_priority_bucket: sign_up_mfa_priority_bucket,
```